ntoskrnl/kdbg/kdb_symbols.c

```diff
@@ -330,15 +330,15 @@ KdbSymProcessSymbols(
     KeSetEvent(&SymbolsToLoadEvent, IO_NO_INCREMENT, FALSE);
 }
 
+
 VOID
```

Comment:
Note: All the following functions (incl. KdbInitialize) should be moved out of kdb_symbols, since they are unrelated in general with symbols management. This will be done elsewhere (not in this PR).